.gitignore

```diff
@@ -1,6 +1,8 @@
 *.deb
 *.swp
-carbon-relay-ng
-carbon-relay-ng.test
+/carbon-relay-ng
```

Comment:
interesting, i get no such notice when i run status after making changes in that dir, but anyway looks good.